Examples/Tests/ohm_solver/EM_modes/PICMI_inputs.py

```diff
@@ -0,0 +1,365 @@
+#!/usr/bin/env python3
+#
+# --- Test script for the kinetic-fluid hybrid model in WarpX wherein ions are
+# --- treated as kinetic particles and electrons as an isothermal, inertialess
+# --- background fluid. The script is set up to produce either parallel or
+# --- perpendicular (Bernstein) EM modes and can be run in 1d, 2d or 3d
+# --- Cartesian geometries. As a CI test only a small number of steps are taken
+# --- using the 1d version.
```

Comment:
For consistency with the other tests, could you also add the Munoz reference here?